aten/src/ATen/ScalarOps.h

```diff
@@ -4,23 +4,32 @@
 #include <ATen/Tensor.h>
 #include <ATen/Functions.h>
 
+namespace at {
+template <typename scalar_t>
+inline void fill_inplace(Tensor& self, Scalar value_scalar) {
+  auto value = value_scalar.to<scalar_t>();
+  scalar_t * dptr = static_cast<scalar_t *>(self.data_ptr());
+  *dptr = value;
+}
```

Comment:
I don't think this needs to live in the header; it is only used from one cpp file right?